operator-common/src/test/resources/END_ENTITY_CERT.pem

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Isn't this the same as what you have in the constants in `MockCertManager`? Why do we keep it twice?

Reply:
I added them to follow the other Cluster & Client CA certs in the `MockCertManager` which exist as pem files under test resources as well.

I didn't add the key store though since it's not loaded or used in the `MockCertManager`.

Reply:
So, is it used anywhere? They do not seem to be referenced in the code.

Reply:
I didn't see them used anywhere. I assumed they're added there for a reason, I can only making them available as files allows folks to check them out with openssl cli easily without having to copy / paste from the IDE. 

do you want me to remove only the ones I added? or all of them?

Reply:
Well, I did not check how the others are used. I guess @ppatierno added them so he might know how they are used. But in any case, if we do not know how and what for are they used I would not add the new ones just because I guess.

Reply:
I went through my PR where I added those .pem files and they are not used at all ... of course it's 3 years ago :-) Tbh I think we can delete them. There is no code where they are loaded as resources but the hard-coded ones are used instead.

Reply:
@scholzj @ppatierno I deleted the ones added in this PR, we can delete the other in a separate PR I believe if you wanted.